### ISSUE TITLE
No detection after  training on VOC datasets

### ISSUE THREAD

**Issue by hongrui16** [pjreddie/darknet#810]
I run the training on the VOC datasets , then i use the generated weight file for the test.
I run the command ''./darknet detector test cfg/voc.data cfg/yolov3-voc.cfg yolov3-voc.weights data/dog.jpg'', but there is no object been detected. It is not a matter of threshold because i change its value to 0.
At first I thought it might be some bugs in the training code as the VOC datasets is frequently being used. I review the code, but got a conclusion in a short time. So I decided to train the YOLO3 with VOC in another version ''https://github.com/experiencor/keras-yolo3''. There has been a weight file generated and I have used it for test. There are some objects been detected, though the results are not precise.

The training label is generated in this commander "cat 2007_train.txt 2007_val.txt 2012_*.txt > train.txt
"
So I wonder the annotations is not been taken in the training progress in Darknet. That is there is no bounding box, so there are no correct weight parameters. While, this wonder need to be proved.

**Comment by hongrui16** [comment]
It was due to the insufficiency of datasets.

**Comment by ljjyxz123** [comment]
Have you solved that problem by retraining the weights with more datasets?

**Comment by hongrui16** [comment]
@ljjyxz123  Yes, I solve the problem with more datasets. It works well.

**Comment by AbhijeetSrivastava96** [comment]
> @ljjyxz123 Yes, I solve the problem with more datasets. It works well.

will u please help me ?
Because after retraining, there is a failure in detection of the object.?
Will you please suggest me
